### DISCUSSION TITLE
x_min_pos and y_min_pos

### DISCUSSION THREAD

**Discussion by AnyMouse835** [Jyers/Marlin#936] [Ideas]
If you install a Micro Swiss Direct Drive on, in my case, an E3V2, the home position of X & Y axis is off the build bed.

I believe from this discussion https://github.com/Jyers/Marlin/discussions/344 that the only way of correcting this currently is to build version of the firmware with the correct settings

Would it be possible to implement setting these values through the UI?

(I've tried using X & Y home offsets, but they don't appear to do anything - maybe not working?)

**Comment by smooreace** [comment]
Microswiss tells you what needs done through compiling because that where
it has to be done.

**Comment by vhristenco** [comment]
@AnyMouse835, run this gcode and you're all set:
```
M206 X-5 Y-10
M500
```

**Reply by AnyMouse835** [reply]
Thanks @vhristenco - you can set home offsets through the UI - they didn't have an effect.

**Reply by vhristenco** [reply]
@AnyMouse835  The UI sets the position by sending a G92, which is not the same thing.

**Reply by AnyMouse835** [reply]
Oh, right, thank you.
